x-pack/test/security_solution_endpoint/apps/integrations_feature_flag/artifact_entries_list.ts

```diff
@@ -0,0 +1,376 @@
+import { FtrProviderContext } from '../../ftr_provider_context';
+import {
+  ArtifactBodyType,
+  getArtifactsListTestsData,
+  ArtifactActionsType,
+  AgentPolicyResponseType,
+  getCreateMultipleData,
+  MultipleArtifactActionsType,
+} from './mocks';
+import { PolicyTestResourceInfo } from '../../services/endpoint_policy';
+import { targetTags } from '../../target_tags';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'artifactEntriesList']);
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+  const endpointArtifactsTestResources = getService('endpointArtifactTestResources');
+  const endpointTestResources = getService('endpointTestResources');
+  const retry = getService('retry');
+  const esClient = getService('es');
+  const supertest = getService('supertest');
+  const find = getService('find');
+  const toasts = getService('toasts');
+  const policyTestResources = getService('policyTestResources');
+  const unzipPromisify = promisify(unzip);
+
+  const removeAllArtifacts = async () => {
+    for (const listId of ENDPOINT_ARTIFACT_LIST_IDS) {
+      await removeExceptionsList(listId);
+    }
+  };
+
+  const removeExceptionsList = async (listId: string) => {
+    await supertest
+      .delete(`${EXCEPTION_LIST_URL}?list_id=${listId}&namespace_type=agnostic`)
+      .set('kbn-xsrf', 'true');
+  };
+
+  describe('For each artifact list under management', function () {
+    targetTags(this, ['@ess', '@serverless']);
+
+    this.timeout(60_000 * 5);
+    let indexedData: IndexedHostsAndAlertsResponse;
+    let policyInfo: PolicyTestResourceInfo;
+
+    before(async () => {
+      indexedData = await endpointTestResources.loadEndpointData();
+    });
+    after(async () => {
+      await endpointTestResources.unloadEndpointData(indexedData);
+    });
+
+    const checkFleetArtifacts = async (
+      identifier: string,
+      expectedArtifact: ArtifactElasticsearchProperties,
+      expectedDecodedBodyArtifact: ArtifactBodyType,
+      policy?: PolicyTestResourceInfo
+    ) => {
+      // Check edited artifact is in the list with new values (wait for list to be updated)
+      let updatedArtifact: ArtifactElasticsearchProperties | undefined;
+      await retry.waitForWithTimeout('fleet artifact is updated', 120_000, async () => {
+        const artifacts = await endpointArtifactsTestResources.getArtifactsFromUnifiedManifestSO();
+
+        // This expects manifest artifact to come from unified so
+        const manifestArtifact = artifacts.find((artifact) => {
+          return (
+            artifact.artifactIds.includes(
+              `${expectedArtifact.identifier}-${expectedArtifact.decoded_sha256}`
+            ) && artifact.policyId === policy?.packagePolicy.id
+          );
+        });
+
+        if (!manifestArtifact) return false;
+
```

Comment:
This part was modified, compared to ../integrations/artifact_entries_list.ts, to properly parse new SO.